src/markdown_generators.js

```diff
@@ -109,7 +109,10 @@ turndownService.addRule("refshortcode", {
     const children = Array.prototype.slice.call(node.childNodes)
     if (!children.filter(child => child.nodeName === "IMG").length > 0) {
       // if this link doesn't contain an image, escape the content
-      content = turndownService.escape(content)
+      // except first make sure there are no pre-escaped square brackets
+      content = turndownService.escape(
+        content.replace(/\\\[/g, "[").replace(/\\\]/g, "]")
+      )
```

Comment:
should we add a quick test for this? could use for instance the link from the economics course that is causing a problem (https://github.com/mitodl/hugo-course-publisher/issues/166)

Reply:
@aliceriot test added